backend/routes/doctorOrders.js

```diff
@@ -8,7 +8,7 @@ const axios = require('axios').default;
 
 const DoctorOrder = require('../models/doctorOrders');
 
-router.patch("/fhir/rems/:id", async (req, res, next) => {
+router.patch("/$process-message/:id", async (req, res, next) => {
```

Comment:
change this to be ([\$])process-message. See https://github.com/expressjs/express/issues/2987 for more context